test.js

```diff
@@ -9,5 +9,8 @@ function printMessage(message) {
 
 function print2Message(message) {
     console.log(message);
-    
+}
+
+function xyz() {
```

Comment:
@manjitp46  Function name should be proper its not like xyz it should be something else